src/index.ts

```diff
@@ -1067,20 +1095,26 @@ export function createForgoInstance(customEnv: any) {
     nodes: ArrayLike<ChildNode>,
     from: number,
     to: number
-  ) {
+  ): ChildNode[] {
+    const justDeletedNodes: ChildNode[] = [];
+
     const nodesToRemove = sliceNodes(nodes, from, to);
     if (nodesToRemove.length) {
       const parentElement = nodesToRemove[0].parentElement as HTMLElement;
       const deletedNodes = getDeletedNodes(parentElement);
       for (const node of nodesToRemove) {
         // If the consuming application has manually mucked with the DOM don't
         // remove things it added
-        if (!getForgoState(node)) continue;
+        const state = getForgoState(node);
```

Comment:
Why prefer this style?

Reply:
In this case, there's no reason to have 'state' as a variable - since it's used only once.
I'll change it.